test/sender_and_receiver_integration_test.exs

```diff
@@ -130,4 +158,29 @@ defmodule SenderAndReceiverIntegrationTest do
       {:ok, %{state | reply_buffer: reply}}
     end
   end
+
+  defmodule TestNoReplySlowDispatcher do
+    require Logger
+
+    @behaviour MLLP.Dispatcher
+
+    def dispatch(:mllp_hl7, message, state) when is_binary(message) do
+      reply =
+        MLLP.Ack.get_ack_for_message(
+          message,
+          :application_accept
+        )
+        |> to_string()
+
+      Process.sleep(1500)
```

Comment:
Don't like doing this, but not sure there's a better way without making more changes. Specifically right now the  reconnect timer  is hard coded at 1000 ms. If we had an option for this, we could avoid this, but I think that should be done in another PR along with other reconnect / time options and behavior. I'm not opposed to doing it here though.

Reply:
I don't see this module being used.

Reply:
Yeah, I started re-working `stop_connection` to make use of maybe_close_socket/1, but  then there was a reason I paused and  said these are not the same cases, but I can look again and re-work it.